### DISCUSSION TITLE
Missing transcript

### DISCUSSION THREAD

**Discussion by yukihana2k3** [openai/whisper#2160] [Q&A]
When transcribing Chinese language, I sometimes encounter missing segments in my transcriptions, typically lasting from 5 to 10 seconds. I've experimented with various models, yet the issue persists; however, the missing segments occur at different points with each model. On average, a 10-minute audio file will contain 1 to 2 of these missing segments. I would greatly appreciate your assistance in resolving this matter. Thank you.
![image](https://github.com/openai/whisper/assets/168797511/f0311af5-76e1-4219-b292-95fd863d3f5d)

**Comment by gongouveia** [comment]
@yukihana2k3 Hi, did you solve this issue?
